### PR TITLE
Create docs for v1.0.x

### PR DESCRIPTION
**Pull request by 6543**
and also fix / enhance some

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-07-21
+## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-07-22
```
Comment (6543) [inline CHANGELOG.md:3]:
```suggestion
## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-07-23
```

File: docs/docs/91-migrations.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 Some versions need some changes to the server configuration or the pipeline configuration files.
 
-## 1.0.0 (next)
+## 1.1.0 (next)
```
Comment (pat-s) [inline docs/docs/91-migrations.md:5]: It's usually unclear what the next version will be - just omit this one?

File: docs/versioned_docs/version-1.0/10-intro.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  a-test-step:
+    image: debian
+    commands:
+      - echo "Testing.."
+```
+
+### Steps are containers
+
+- Define any container image as context
+  - either use your own and install the needed tools in custom image or
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/10-intro.md:29]:
```suggestion
  - either use your own and install the needed tools in a custom image
```

File: docs/versioned_docs/version-1.0/10-intro.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  - search for available images that are already tailored for your needs on container registries like [Docker Hub](https://hub.docker.com/search?type=image)
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/10-intro.md:30]:
```suggestion
  - or search for available images that are already tailored for your needs in image registries like [Docker Hub](https://hub.docker.com/search?type=image)
```

File: docs/versioned_docs/version-1.0/10-intro.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- List the commands that should be executed in your container, in order to build or test your application
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/10-intro.md:31]:
```suggestion
- List the commands that should be executed in the container
```

File: docs/versioned_docs/version-1.0/10-intro.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+-   image: debian
++   image: mycompany/image-with-awscli
+    commands:
+      - aws help
+```
+
+### File changes are incremental
+
+- Woodpecker clones the source code in the beginning
+- Changes to files are persisted through steps as the same volume is mounted to all steps
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/10-intro.md:45]:
```suggestion
- File changes are persisted throughout individual steps as the same volume is being mounted in all steps
```

File: docs/versioned_docs/version-1.0/20-usage/10-intro.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+# Webhooks
+
+When you activate your repository Woodpecker automatically add webhooks to your forge (e.g. GitHub). There is no manual configuration required.
+
+Webhooks are used to trigger pipeline executions. When you push code to your repository, open a pull request, or create a tag, your forge will automatically send a webhook to Woodpecker which will in turn trigger pipeline execution.
+
+## Configuration
+
+To configure your pipeline you should place a `.woodpecker.yml` file in the root of your repository. The .woodpecker.yml file is used to define your pipeline steps. It is a superset of the widely used docker-compose file format.
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/20-usage/10-intro.md:25]:
```suggestion
To configure your pipeline you must create a `.woodpecker.yml` file in the root of your repository or in a directory named `.woodpecker`. The .woodpecker.yml file is used to define your pipeline steps. It is a superset of the widely used docker-compose file format.
```

File: docs/versioned_docs/version-1.0/10-intro.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
+# Welcome to Woodpecker
+
+Woodpecker is a simple CI engine with great extensibility. It runs your pipelines inside [containers](https://opencontainers.org/), so if you are already using them in your daily workflow, you'll love Woodpecker for sure.
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/10-intro.md:3]:
```suggestion
Woodpecker is a simple CI engine with great extensibility. It focuses on executing pipelines inside [containers](https://opencontainers.org/). If you are already using containers in your daily workflow, you'll for sure love Woodpecker.
```

File: docs/versioned_docs/version-1.0/20-usage/10-intro.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
+# Getting started
+
+## Repository Activation
+
+To activate your project navigate to your account settings. You will see a list of repositories which can be activated with a simple toggle. When you activate your repository, Woodpecker automatically adds webhooks to your forge (e.g. GitHub).
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/20-usage/10-intro.md:5]:
```suggestion
To activate your project navigate to your account settings. You will see a list of repositories which can be activated with a simple toggle. When you activate your repository, Woodpecker automatically adds webhooks to your forge (e.g. GitHub or Gitea).
```

File: docs/versioned_docs/version-1.0/20-usage/10-intro.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+Webhooks are used to trigger pipeline executions. When you push code to your repository, open a pull request, or create a tag, your forge will automatically send a webhook to Woodpecker which will in turn trigger pipeline execution.
```
Comment (pat-s) [inline docs/versioned_docs/version-1.0/20-usage/10-intro.md:7]:
```suggestion
Webhooks are used to trigger pipeline executions. When you push code to your repository, open a pull request, or create a tag, your forge will automatically send a webhook to Woodpecker which will in turn trigger the pipeline execution.
```

File: docs/docs/91-migrations.md (hunk 2 above)
Comment (6543) [inline docs/docs/91-migrations.md:5]:
```suggestion
## next
```

Comment (6543) [reply docs/docs/91-migrations.md]: Changed to 'next' only

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-07-21
+## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-07-23
```
Comment (6543) [inline CHANGELOG.md:3]:
```suggestion
## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-07-25
```

**Review by pat-s — APPROVED** [review]

**Pull request merged**